### PR TITLE
[Changed] the storytelling of the challenge

### PR DESCRIPTION
**Pull request by stanete**
## Description 🎩

After receiving some feedback about our backend challenge we arrived at the conclusion that the storytelling could be improved.

I tried to give some `tips` about our engineering culture and what we value without being too specific. I also added an `out of scope` section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
++ If the payment is for an ordinary book, you must generate a `shipping label` with a notification that it is a tax-exempt item.
++ If the payment is for any digital media (music, video), in addition to sending the description of the purchase by email to the buyer, grant a discount voucher of 10% to the buyer associated with the payment.
 
-Since the proposal does not require a final working code, the use of libraries is allowed to facilitate the implementation.
+## Tips
 
-Please describe your modifications and reasoning in a separate text file or markdown.
+* Feel free to modify/refactor the bootstrap files if you think it's necessary.
+* Don’t over-engineer your solution! Your Extremme Programming team loves clean and cohesive code, so try to keep it simple.
+* It’s ok if you use third-party libraries, but be aware that your team prefers a vanilla solution.
```
Comment (joseahernandez) [inline README.md:20]: For me that sentence is like say don't use third-party libraries.  I would leave the sentence as `It’s ok if you use third-party libraries` and then we can ask the candidate why he/she has used it.

Comment (JaiL81) [reply README.md:20]: I agree, I prefer that the candidate uses what he/she feels more productive.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,30 @@
-# backend-challenge-vlc
-Backend challenge for Creditas Valencia
+# A small journey in an eXtremme Programming team
 
-You have been assigned to prototype a new version of an existing payment stream which is fragile and not scalable, since it requires large modifications whenever a new item is sent or removed.
+This is the Backend challenge for Creditas Valencia.
 
-The new prototype should be able to handle the following scenarios:
+You are part of an [Extreme Programming](http://www.extremeprogramming.org/) team. You all share [some values](http://www.extremeprogramming.org/values.html) and have agreed to the same [rules](http://www.extremeprogramming.org/rules.html).
```
Comment (JaiL81) [inline README.md:5]: I like this way of saying that we want test without saying it specifically. If the candidate reads the rules of XP and does not write them, then it is a red flag.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,30 @@
...
+* The challenge **does not require a final working code necessarily**. We want to understand how you solve problems. If you have any questions or doubts, feel free to ask.
+* Describe your modifications and reasoning in a separate markdown file.
+* What language? Kotlin, Ruby, Python, Java or PHP.
 
-What language? Kotlin, Ruby, Python, Java or PHP
 
+## Out of scope
+
+* **It is not necessary** to create the implementations for sending emails, to print the shipping label, etc. For these cases (email, shipping label) **only create method calls**, to indicate that it would be the place where the sending would take place.
+* **It is not necessary** to use a databae. Although in real life you would, for the challenge, in-memory data is enough.
```
Comment (JaiL81) [inline README.md:29]:
```suggestion
* **It is not necessary** to use a database. Although in real life you would, for the challenge, in-memory data is enough.
```

File: README.md (hunk 1 above)
Comment (stanete) [reply README.md:20]: Agree!

**Pull request merged**